PacketDelivery-VC-ReferenceExample/data-consumer/keycloak/values.yaml

```diff
@@ -0,0 +1,97 @@
+route:
+  enabled: true
+  host: kc-consumer.domain.org
```

Comment:
If you do not configure this, it gets a default route. You can also leave out the tls and certificate part then. This makes the installation more generic, e.g. everyone can use it(at least in openshift) without setting up  external-dns, cert-manager and cert-utils.